### PR TITLE
PLATUI-2674: Converted play-nunjucks to multi-project build, removed …

### PR DESCRIPTION
**Pull request by JoPintoPaul**
…Play 2.6 and Play 2.7, added Play 2.9 and Play 3.0

### CODE REVIEW

File: build.sbt
Comment (oscarduignan) [inline build.sbt:22]: @JoPintoPaul do we need the routes stuff ? For play-frontend-hmrc and play-language we did because they both have routers but play-nunjucks doesn't have any compiled routers

Comment (JoPintoPaul) [reply build.sbt:22]: @oscarduignan LET'S FIND OUT *moves fast and breaks things*

File: project/plugins.sbt
Comment (oscarduignan) [inline project/plugins.sbt:16]: not worth doing anything more now - but one thing that I'm still not sure of with all this is whether play-nunjucks itself needs to have a dependency on play itself - or if this is just here because of the integration projects

for example, we still have to run the build multiple times with different PLAY_VERSIONs in play-frontend-hmrc because we have routes that needs to be compiled and generate reverse routes - however bootstrap refactored their compiled routes to just be a router (because they didn't need reverse routing) and so they can just run the build once and all the different versions of the library get built at the same time for different versions of play because they don't depend on play plugin. I wonder if play-nunjucks could go further to remove the PLAY_VERSION stuff because it seems like it might just be using the play plugin for the integration sub projects - but that's total "future" stuff

**Comment by oscarduignan** [comment]
other than that one bit - looks really good to me 🙌 , I'm sure there's always more that we can simplify with sbt config because there's loads that I just kind of accept as needed because it's too costly to bottom out exactly how it's being used - but we just want to unblock people upgrading play now rather than improve the build for future maintainers

**Review by oscarduignan — DISMISSED** [review]

File: project/plugins.sbt
Comment (JoPintoPaul) [reply project/plugins.sbt:16]: @oscarduignan That's some really good thoughts. I think it's worth capturing these on our upgrading notes document.

File: README.md
Comment (oscarduignan) [inline README.md:244]: @JoPintoPaul just thought, one thing it might be worth adding - a note to say prominently something like  "The latest major version releases have not yet been performance tested, please make sure when updating that you check for regressions before going to prod."

**Review by oscarduignan — APPROVED** [review]

File: README.md
Comment (JoPintoPaul) [reply README.md:244]: @oscarduignan That's excellent advice, do you think that would perhaps be better in the team channels? Wonder if the README will quickly become out-of-date otherwise.

**Review by oscarduignan — APPROVED** [review]

**Pull request merged**
